Update to 2.6.3

isuruf opened the pull request:
Fixes https://github.com/conda-forge/flex-feedstock/issues/11

isuruf committed:
Update to 2.6.3

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jakirkham commented:
Also please enable CircleCI on your fork and repush.

jakirkham commented:
I think the failure is caused by the dropping of [PIC support]( https://github.com/westes/flex/blob/v2.6.3/NEWS#L35-L37 ) in `2.6.3`. Feel free to just drop the file tests for these PIC libraries.

jakirkham commented:
Added the fixes note in the OP to close out the related issue (on merge). Hope that is ok.

isuruf committed:
Drop PIC version library checking

PIC version libraries are dropped now. https://github.com/westes/flex/blob/v2.6.3/NEWS#L35-L37

jakirkham commented:
Thanks @isuruf . LGTM.
